contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -388,6 +389,17 @@ check_tsql_version(char **newval, void **extra, GucSource source)
 	return true;
 }
 
+static bool
+check_endpoint(char **newval, void **extra, GucSource source)
+{
+	if (sql_dialect == SQL_DIALECT_TSQL)
```

Comment:
Let's IS_TDS_CLIENT() function if the requirement is to block from TDS Endpoint.